src/components/Announcement/index.tsx

```diff
@@ -0,0 +1,181 @@
+
+export interface AnnouncementProps {
+	open?: boolean;
+	onClose: (
+		event: React.SyntheticEvent<any> | Event,
+		reason: AnnouncementCloseReason,
+	) => void;
+	campaignId: string;
+	children: React.ReactNode;
+	illustration?: React.ReactNode;
+	title: string;
+	linkHref: string;
+	linkLabel: string;
+	sx?: SxProps;
+}
+
+const AnnouncementRoot = styled(Box, {
+	name: 'Announcement',
+	slot: 'root',
+})<{ ownerState: AnnouncementProps }>(({ theme, ownerState }) => ({
+	alignItems: 'center',
+	backgroundColor: 'white',
+	borderRadius: shape.border_radius.md.value + 'px',
+	boxShadow: '0px 0px 8px 0px rgba(35, 68, 94, 0.3)', // TODO add a design token for box shadows
+	display: 'flex',
+	gap: theme.spacing(3),
+	maxWidth: '100%',
+	padding: theme.spacing(3),
+	width: '100%',
+}));
+
+const AnnouncementIllustration = styled('figure', {
+	name: 'Announcement',
+	slot: 'illustration',
+})(({ theme }) => ({
+	margin: 0,
+	maxWidth: 48,
+	minWidth: 32,
+	[theme.breakpoints.up('sm')]: {
+		maxWidth: 72,
+	},
+	'& *': {
+		width: '100%',
+	},
+}));
+
+const AnnouncementTitle = styled('h3', {
+	name: 'Announcement',
+	slot: 'title',
+})(({ theme }) => ({
+	...theme.typography.bodyLg,
+	margin: 0,
+	display: 'flex',
+	alignItems: 'center',
+}));
+
+const AnnouncementContent = styled(Stack, {
+	name: 'Announcement',
+	slot: 'content',
+})(({ theme }) => ({
+	flexGrow: 1,
+	gap: theme.spacing(1),
+	[theme.breakpoints.up('sm')]: {
+		gap: theme.spacing(2),
+	},
+}));
+
+/**
+ * This component is a notification intended to be displayed in the bottom right
+ * corner of the screen, usually sharing updates on the product with our users.
+ *
+ * This is a Themed Component as documented here: https://mui.com/material-ui/customization/creating-themed-components/.
+ * As such, it can be modified in the theme.
+ */
+export const Announcement = React.forwardRef<HTMLDivElement, AnnouncementProps>(
+	function Announcement(inProps, ref) {
+		const props = useThemeProps({ props: inProps, name: 'Announcement' });
+		const {
+			illustration,
+			campaignId,
+			children,
+			title,
+			linkHref,
+			linkLabel,
+			onClose,
+			open,
+			sx,
+			...other
+		} = props;
+		const ownerState = {
+			...props,
+			campaignId,
+			children,
+			illustration,
+			title,
+			linkHref,
+			linkLabel,
+			onClose,
+			open,
+		};
+		const { state: analytics } = useAnalyticsContext();
+
+		// TODO remove this event after the first campaign, when we are sure that the notification is being shown properly
+		useEffect(() => {
+			analytics.webTracker?.track('Announcement: open', { campaignId });
+		}, []);
+
+		return (
+			<Snackbar
+				open={open}
+				anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+				TransitionComponent={(props: SlideProps) => {
+					return <Slide direction="up" {...props} />;
+				}}
+				ref={ref}
+				onClose={onClose}
+				sx={[
+					(theme) => ({ maxWidth: 350, zIndex: theme.zIndex.drawer - 1 }),
```

Comment:
Just curious, how did you settle on this specific zIndex? It may be good to leave a comment above this line so that we don't forget/lose the context

Reply:
I wanted to prevent the component showing on top of the support menu. The default z-index for snackbars is actually very high, only `tooltip` is higher.